docs/source/pythonapi/examples/mgxs-part-ii.ipynb

```diff
@@ -34,8 +34,16 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/usr/local/lib/python2.7/dist-packages/IPython/kernel/__main__.py:11: QAWarning: pyne.rxname is not yet QA compliant.\n",
-      "/usr/local/lib/python2.7/dist-packages/IPython/kernel/__main__.py:11: QAWarning: pyne.ace is not yet QA compliant.\n"
+      "/usr/local/lib/python2.7/dist-packages/matplotlib-1.5.1+1178.ga40c9ec-py2.7-linux-x86_64.egg/matplotlib/__init__.py:884: UserWarning: axes.color_cycle is deprecated and replaced with axes.prop_cycle; please use the latter.\n",
+      "  warnings.warn(self.msg_depr % (key, alt_key))\n",
+      "/usr/local/lib/python2.7/dist-packages/matplotlib-1.5.1+1178.ga40c9ec-py2.7-linux-x86_64.egg/matplotlib/__init__.py:1362: UserWarning:  This call to matplotlib.use() has no effect\n",
+      "because the backend has already been chosen;\n",
+      "matplotlib.use() must be called *before* pylab, matplotlib.pyplot,\n",
+      "or matplotlib.backends is imported for the first time.\n",
```

Comment:
Also interestingly enough the plots do not show up inline for mgxs-part-ii.ipynb and mgxs-part-iii.ipynb unless `%matplotlib inline` appears *after* the `openmoc` imports. It does work just fine, however, for our other notebooks, *but* the plot formatting is different than what is shown in our current documentation.